alacritty/src/display/mod.rs

```diff
@@ -188,9 +188,18 @@ pub struct Display {
 
 impl Display {
     pub fn new<E>(config: &Config, event_loop: &EventLoop<E>) -> Result<Display, Error> {
-        // Guess DPR based on first monitor.
-        let estimated_dpr =
-            event_loop.available_monitors().next().map(|m| m.scale_factor()).unwrap_or(1.);
+        #[cfg(any(feature = "wayland", target_os = "macos", windows))]
+        #[allow(unused_variables)]
```

Comment:
Why is this necessary? Our cfg annotation should be able to tell when we need this variable.